lotus/client_test.go

```diff
@@ -21,7 +21,6 @@ func TestMain(m *testing.M) {
 }
 
 func TestClientVersion(t *testing.T) {
```

Comment:
Despite we could run ldevnets in parallel for tests (which is nice), they're quite intensive. 
Don't be so bad and run them serially.